.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### Milestone (Odoo version)
+- 
```

Comment:
It doesn't seem to be necessary. doesn't it ? see preview : 
https://github.com/grap/maintainer-tools/blob/github_template/.github/ISSUE_TEMPLATE.md